voice/bxml/callbacks/transferComplete.md

```diff
@@ -1,7 +1,8 @@
 {% method %}
 ##  Transfer Complete Event – <Transfer> verb
-This event is sent to the callbackUrl of the Leg-A call when the transferred call(B-leg) completes.
-In a simultaneous ringing scenario, only one call leg-B succeeds and this event corresponds to that successful leg. If none of the calls were answered, transferComplete is not sent.
+This event is sent to the `transferCompleteUrl` of the A-leg's `<Transfer>` verb when the transferred call (B-leg) completes.
+In a simultaneous ringing scenario, only one B-leg succeeds and this event corresponds to that successful leg.
+If none of the calls were answered, the `transferComplete` event corresponds to one of the legs.
```

Comment:
Could you please also fix the example below? (line ~53)
It says `"cause"            : "completed",` but that is incorrect. The cause should be `hangup`.